src/Common/Core/Impl/Idle/IdleTimeAction.cs

```diff
@@ -4,10 +4,8 @@
 using System;
 using System.Collections.Generic;
 using Microsoft.Common.Core.Shell;
-using Microsoft.Languages.Core.Utility;
-using Microsoft.Languages.Editor.Shell;
 
-namespace Microsoft.Languages.Editor.Tasks {
+namespace Microsoft.Common.Core.Idle {
```

Comment:
I don't technically need it in common but it is not tied to the editor either. Common is not just for UI-less apps anymore, it contains plenty of UI definitions and concept of the main thread and locale.